base/range.jl

```diff
@@ -374,7 +374,7 @@ julia> step(range(2.5, stop=10.9, length=85))
 ```
 """
 step(r::StepRange) = r.step
-step(r::AbstractUnitRange{T}) where{T} = oneunit(T)
+step(r::AbstractUnitRange{T}) where{T} = one(T) - zero(T)
```

Comment:
Should be `oneunit(T) - zero(T)`, no?

Reply:
Ah yes, I mixed them up and thought that `oneunit` was the multiplicative identity.